vertical-pod-autoscaler/e2e/vendor/k8s.io/autoscaler/vertical-pod-autoscaler/pkg/utils/status/status_updater.go

```diff
@@ -53,7 +53,7 @@ func (su *Updater) Run(stopCh <-chan struct{}) {
 				return
 			case <-time.After(su.updateInterval):
 				if err := su.client.UpdateStatus(); err != nil {
-					klog.Errorf("Status update by %s failed: %v", su.client.holderIdentity, err)
+					klog.ErrorS(err, "Status update failed", "holderIdentity", su.client.holderIdentity)
```

Comment:
Please don't change any vendor directories manually. Those should be updated via go mod vendor.

Reply:
right sorry